### PR TITLE
plans/models: avoid accessing topic choices with wrong keys

### PR DESCRIPTION
**Pull request by Rineee**

### CODE REVIEW

File: meinberlin/apps/plans/models.py [hunk 1]
```diff
@@ -129,7 +129,14 @@ def administrative_district(self):
     def topic_names(self):
         if hasattr(settings, 'A4_PROJECT_TOPICS'):
             choices = dict(settings.A4_PROJECT_TOPICS)
-            return [choices[topic] for topic in self.topics]
+            names = []
+            for topic in self.topics:
+                try:
+                    name = choices[topic]
+                except:
+                    name = topic
```
Comment (Rineee) [inline meinberlin/apps/plans/models.py:137]: or pass?

Comment (fuzzylogic2000) [reply meinberlin/apps/plans/models.py:137]: Hmm. I like this one!

**Review by fuzzylogic2000 — APPROVED** [review]

File: meinberlin/apps/plans/models.py (hunk 1 above)
Comment (fuzzylogic2000) [reply meinberlin/apps/plans/models.py:137]: Oh right! The linter complains and is absolutely right! You shouldn't just use try without the actual error. So, could you please add that?

**Comment by Rineee** [comment]
In the multiselectfield code I saw, that there actually is this get function for dictionaries that handles key errors :)

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! Good, that you had a look!

**Pull request merged**